src/targz_sizes.c

```diff
@@ -137,7 +159,11 @@ int main(int argc, char** argv) {
     inflateGetHeader(&gzip_stream, &gzip_header);
 
     // state machine
+    int file_blocks = 0;
     int skip_blocks = 0;
+    int read_blocks = 0;
+    unsigned char* long_filename = (unsigned char*) filename_buff;
```

Comment:
And I've struggled a bit with this one.  Anyway, I've addressed the comment and made other (what seem to be appropriate) corresponding updates. Addressed in commit: 38baeb79a0eb820299e0411fad1cea7b263f96e2